src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3140,6 +3140,17 @@ jint Arguments::finalize_vm_init_args(bool patch_mod_javabase) {
     DynamicDumpSharedSpaces = true;
   }
 
+  if (AutoCreateSharedArchive) {
+    if (SharedArchiveFile == NULL) {
+      log_info(cds)("-XX:+AutoCreateSharedArchive must work with a valid SharedArchiveFile");
```

Comment:
The log message should be consistent with other existing messages. E.g., 

```
 fatal("-XX:+VerifyHeavyMonitors requires -XX:+UseHeavyMonitors");
```

So we should have:

```
 -XX:+AutoCreateSharedArchive requires -XX:SharedArchiveFile
```